ui-tests/src/test/java/org/openmrs/reference/page/HeaderPage.java

```diff
@@ -27,8 +27,8 @@ public void clickOnHomeLink() {
     }
 
     public void logOut() throws InterruptedException {	
-	 WebDriverWait waiter = new WebDriverWait(driver, 30);
-	 waiter.until(ExpectedConditions.visibilityOfElementLocated(By.className("logout")));
+	WebDriverWait waiter = new WebDriverWait(driver, 30);
+	waiter.until(ExpectedConditions.visibilityOfElementLocated(By.className("logout")));
```

Comment:
Handle the format issues

Reply:
use `LOGOUT` field rather than `By.className("logout")`